typescript/packages/well-completions-plot/package.json

```diff
@@ -0,0 +1,29 @@
+{
+    "name": "@webviz/well-completions-plot",
+    "version": "0.0.0",
+    "description": "",
+    "main": "dist/index.js",
+    "types": "dist/index.d.ts",
+    "files": [
+        "/dist"
+    ],
+    "scripts": {
+        "transpile": "tsc --project ./tsconfig.json",
+        "copy-files": "copyfiles package.json dist && copyfiles README.md dist && copyfiles --exclude **/*.{tsx,jsx,js,ts,snap} --up 1 \"src/**/*\" dist/",
```

Comment:
please use the same template than other packages.
package.json and README.md are automatically copied (no need here), and former copyfile lead to duplicated code at the wrong place in the generated package :/